pages/dkp/kommander/1.4/projects/index.md

```diff
@@ -8,27 +8,31 @@ excerpt: Multi-cluster Configuration Management
+A Project Namespace is the grouping of all the individual standard Kubernetes namespaces that make up the concept of a Project Namespace. Project Namespaces isolate configurations across clusters. Individual standard Kubernetes namespaces are automatically created on all clusters belonging to the project. When creating a new project, you can customize the Kubernetes namespace name that is created. A Project Namespace is a Kommander specific concept.
 
 ## Create a Project
-When you create a Project, you must specify a Project Name, a Namespace Name (optional) and a way to allow Kommander to determine which Kubernetes clusters will be part of this project.
 
-As mentioned previously, a Project Namespace corresponds to a Kubernetes Federated Namespace. By default, the name of the namespace is auto-generated based on the project name (first 57 characters) plus 5 unique alphanumeric characters. You can specify a namespace name, but you must ensure it does not conflict with any existing namespace on the target Kubernetes clusters, that will be a part of the Project.
+When you create a Project, you must specify a Project Name, a Namespace Name (optional) and a way for Kommander to determine which Kubernetes clusters are part of this project.
 
-To determine which Kubernetes clusters will be part of this project, you can either select manually existing clusters or define labels that Kommander will use to dynamically add clusters. The latter is recommended because it will allow you to deploy additional Kubernetes clusters later and to have them automatically associated with Projects based on their labels.
+A Project Namespace corresponds to a Kubernetes Federated Namespace. By default, the name of the namespace is auto-generated based on the project name (first 57 characters) plus 5 unique alphanumeric characters. You can specify a namespace name, but you must ensure it does not conflict with any existing namespace on the target Kubernetes clusters, that will be a part of the Project.
 
-To create a Project, you can either use the Kommander UI or create a Project object on the Kubernetes cluster where Kommander is running (using kubectl or the Kubernetes API). The latter allows you to configure Kommander resources in a declarative way. It’s available for all kinds of Kommander resources.
+To determine which Kubernetes clusters will be part of this project, you can either select manually existing clusters or define labels that Kommander uses to dynamically add clusters. The latter is recommended because it allows you to deploy additional Kubernetes clusters later and have them automatically associated with Projects based on their labels.
 
+To create a Project, you can use the Kommander UI or create a Project object on the Kubernetes cluster where Kommander is running using kubectl or the Kubernetes API. Using the latter method you can configure Kommander resources in a declarative way and it is available for all kinds of Kommander resources.
```

Comment:
```suggestion
To create a Project, you can use the Kommander UI or create a Project object on the Kubernetes cluster where Kommander is running using kubectl or the Kubernetes API. Using the latter method, you can configure Kommander resources in a declarative way.   The declarative method is available for all kinds of Kommander resources.
```
However, I don't really understand what "all kinds of Kommander resources" means here.  Does it work for *all* Kommander resources? (I think it does).   Do we have some examples we could give here to make this more concrete?

i.e. can we say something like:  "The declarative method is available for all Kommander resources, such as 'x', and 'y', and z".  ???

Reply:
@gracedo or @cgawron1 can you guys give me some examples, etc to address this?

Reply:
Other resources - workspaces, kommanderclusters (this is probably too confusing to mention/get into here), projects - maybe those are it 🤔 @TattdCodeMonkey probably has a better idea. But I feel like we can also just remove that sentence entirely. Not really relevant in the Projects section